_deployment/checklist/012_common_configuration.md

```diff
@@ -0,0 +1,270 @@
+---
+title: Common Configuration Steps
+permalink: "deployment/checklist/commonconfiguration.html"
+layout: "document"
+categories: ["deployment", "checklist", "tds"]
+---
+
+# Common TDS Configuration Steps
+
+The following are instructions for steps for common configuration changes in TDS.
+
+## Changing Assessment Labels in TDS
+Assessment labels are defined in the `configs.client_testproperties` table. The following steps can be taken
+to update the label for a particular assessment id:
+
+1. Update the `label` column in the `config.client_testproperties` 
+```
+UPDATE configs.client_testproperties 
+SET label = '<desired assessment label>' 
+WHERE testid = '<assessment id>'; 
+``` 
+2. Flush the redis cache
+3. Delete the currently running `tds-student-war` and `tds-assessment-service` pods and allow Kubernetes to recreate new pods.
+
+
+## Enabling Segment Entry/Exit Approval
+Segment entry/exit approval is disabled by default for all segments. When enabled, a student will need to be approved by a 
+proctor before entering or exiting the segment. Segment entry/exit approval can be enabled for a particular segment by 
+taking the following actions:
+
+1. Update the `entryapproval` or `exitapproval` flag for the particular segment id
+```
+UPDATE configs.client_segmentproperties 
+SET entryapproval = 1 
+WHERE segmentid = '<segment id>';
+```
+2. Flush the redis cache
+3. Delete the currently running `tds-student-war` and `tds-config-service` pods and allow Kubernetes to recreate new pods.
```

Comment:
I do not think database configuration should result in any pods needing to be bounced or deleted.  Same comment for anywhere this is noted.